playground/backend/internal/fs_tool/path_checker.go

```diff
@@ -46,15 +46,15 @@ func isCorrectExtension(filePath string, correctExtension string) bool {
 }
 
 // CheckPathIsValid checks that the file exists and has a correct extension
-func CheckPathIsValid(args ...interface{}) error {
+func CheckPathIsValid(args ...interface{}) (bool, error) {
 	filePath := args[0].(string)
 	correctExtension := args[1].(string)
 	notExists := isNotExist(filePath)
 	if notExists {
-		return fs.ErrNotExist
+		return false, fs.ErrNotExist
 	}
 	if !isCorrectExtension(filePath, correctExtension) {
-		return &WrongExtension{fmt.Sprintf("expected extension %s", correctExtension)}
+		return false, &WrongExtension{fmt.Sprintf("expected extension %s", correctExtension)}
 	}
-	return nil
+	return true, nil
```

Comment:
As I understand this method always returns `(false, error)` or `(true, nil)`. We could return `error` only instead of `(false, error)` and `nil` only instead of `(true, nil)`.

Reply:
No, we can't because all validators have the same signature. 
Other validators need to return bool and error.